components/chat/index.js

```diff
@@ -875,53 +888,82 @@ const Chat = ({
       setTextAnimationIndex(loadingIndex);
       setChatHistory(newHistory);
 
-      const handleResponse = (response, modelType, chatHistoryID) => {
+      const handleResponse = (response, modelType) => {
         console.log("handle response", response);
         if (loadingIndex != -1) {
           const updatedItem = [...newHistory[loadingIndex]];
           updatedItem[1].role = "assistant";
           updatedItem[1].content = response.data.data;
           updatedItem[1].type = modelType;
-
           const updatedHistory = [...newHistory];
           updatedHistory[loadingIndex] = updatedItem;
-
           setChatHistory(updatedHistory);
-
-          if (chatHistoryID == "") {
-            const _idx = historySideData.findIndex((x) => x.id == -1);
-            if (_idx > -1) {
-              historySideData.splice(_idx, 1);
-            }
-            setHistorySideData([
-              {
-                id: response.data.id,
-                title: value,
-                bot: response.data.data,
-                date: response.data.date,
-                thumbnail_url: response.data.thumbnail_url,
-              },
-              ...historySideData,
-            ]);
-            setChatTitle(value);
-          }
-          setChatHistoryID(response.data.id);
+          // const _idx = historySideData.findIndex((x) => x.id == chatHistoryID);
```

Comment:
@abdulsaboorpk why u comment this?

Reply:
Because after the latest changes we don't need it here. Can you open this complete file and verify?

Reply:
If new chat functionality not disturbing then ok for me . Please confirm